### PR TITLE
[MeshingApplication] Fix check to use KratosGlobals and other fixes

### PR DESCRIPTION
**Pull request by loumalouomega**
This uses KratosGlocals instead of manual check. Additionally correcting some checks

### CODE REVIEW

File: applications/MeshingApplication/python_scripts/mmg_process.py [hunk 1]
```diff
@@ -548,15 +548,16 @@ def __generate_variable_list_from_input(self,param):
       if len(self.main_model_part.Nodes) > 0:
           node = (self.main_model_part.Nodes)[1]
           for i in range( 0,param.size()):
```
Comment (philbucher) [inline applications/MeshingApplication/python_scripts/mmg_process.py]: I recommend using
GetStringArray
(Was recently added)

Comment (loumalouomega) [reply applications/MeshingApplication/python_scripts/mmg_process.py]: Done

File: applications/MeshingApplication/python_scripts/mmg_process.py [hunk 2]
```diff
@@ -535,45 +546,45 @@ def __generate_submodelparts_list_from_input(self,param):
...
-              else:
-                  variable_list.append( KratosMultiphysics.KratosGlobals.GetVariable( param[i].GetString()+"_X" ))
-                  variable_list.append( KratosMultiphysics.KratosGlobals.GetVariable( param[i].GetString()+"_Y" ))
-                  if self.domain_size == 3:
-                      variable_list.append( KratosMultiphysics.KratosGlobals.GetVariable( param[i].GetString()+"_Z" ))
-
-      return variable_list
+        return variable_list
 
     def __generate_internal_variable_list_from_input(self,param):
```
Comment (philbucher) [inline applications/MeshingApplication/python_scripts/mmg_process.py]: yet another copy-paste => #5244

Comment (loumalouomega) [reply applications/MeshingApplication/python_scripts/mmg_process.py]: I will update after merge

Comment (loumalouomega) [reply applications/MeshingApplication/python_scripts/mmg_process.py]: Done

**Review by AlejandroCornejo — APPROVED** [review]
I assume @philbucher is satisfied

**Pull request merged**